include/mxnet/c_api.h

```diff
@@ -201,6 +201,12 @@ MXNET_DLL const char *MXGetLastError();
  * \return 0 when success, -1 when failure happens.
  */
 MXNET_DLL int MXRandomSeed(int seed);
+/*!
+ * \brief Initialise MXNet
+ * This ensures the engine has been initialised and
+ * the operators have been registered.
+ */
+MXNET_DLL void MXInitialise();
```

Comment:
In which case is it required to call this method? It sounds like this should be documented widely.

Reply:
Under the hood it just calls the list symbols method which registers the operators, but this means the purpose of the call is clearer.

Reply:
nit: Initialize is spelled with a 'z' :)